EXPO/screens/DrawerScreen.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react';
+import {NavigationActions} from 'react-navigation';
+import PropTypes from 'prop-types';
+import {ScrollView, Text, View, Image} from 'react-native';
+import { DrawerActions } from 'react-navigation';
+//import styles from '../../styles/index';
+import {StyleSheet, Platform} from 'react-native';
+import LinksScreen from "../screens/LinksScreen";
+import HomeScreen from "../screens/Notifications";
+import NearByHotel from "../screens/NearByHotel"
+
+class DrawerScreen extends Component {
+  navigateToScreen = (route) => () => {
+    const navigateAction = NavigationActions.navigate({
+      routeName: route
+    });
+    this.props.navigation.dispatch(navigateAction);
+    this.props.navigation.dispatch(DrawerActions.closeDrawer())
+  }
+
+  render () {
+    return (
+      <View>
+        <ScrollView>
+          <View>
+            <View style={styles.menuItem}>
+              <Text onPress={this.navigateToScreen('HomeScreen')}>
+                Home
+              </Text>
+            </View>
+            <View style={styles.menuItem}>
+              <Text onPress={this.navigateToScreen('LinkScreen')}>
+              Link
+              </Text>
+            </View>
+            <View style={styles.menuItem}>
+              <Image source={{uri:'https://img.icons8.com/ios/60/000000/google-nearby.png'}}/>
+              <Text onPress={this.navigateToScreen('NearByHotel')}>
+                Near By Restaurants
+              </Text>
+            </View>
+          </View>
+        </ScrollView>
+      </View>
+    );
+  }
+}
+
+DrawerScreen.propTypes = {
+  navigation: PropTypes.object
+};
+
+export default DrawerScreen;
+
+const styles = StyleSheet.create({
```

Comment:
we can move this to separate file `DrawerScreen.style.js` and export from there.
I recommend to create directory with name `DrawerScreen` under that all the files related to that will exist.
eg. `DrawerScreen.jsx`(component), `DrawerScreen.test.js`(test cases) & `DrawerScreen.style.js`(stylesheets)